tests/create_saml/main.tf

```
@@ -14,17 +14,21 @@ resource "random_string" "this" {
 }
 
 data "template_file" "this" {
-  template = file("${path.module}/template/metadata.xml")
+  template = file("${path.module}/.fixtures/metadata.xml")
 
   vars = {
     entity_id = random_string.entity_id.result
     domain    = random_string.this.result
   }
 }
 
-module "create_provider" {
+module "create_saml" {
   source = "../../"
 
-  saml_provider_name     = "tardigrade-provider-${random_string.this.result}"
-  saml_provider_metadata = file("${path.module}/template/metadata.xml")
+  iam_identity_provider = {
+    saml = {
+      metadata_document = file("${path.module}/.fixtures/metadata.xml")
```

Relative path here also since we moved the file

```suggestion
      metadata_document = file("${path.module}/../.fixtures/metadata.xml")
```

---

Updated path